docs mvp (#456)
grafana/explore-logs: docs/sources/labels-and-detected-fields/_index.md

---
description: Learn how breaking logs down by Labels and Detected fields can help you find the signal in the noise.
keywords:
  - Logs
  - Explore
  - Labels
  - Analysis
menuTitle: Labels and detected fields
title: Labels and detected fields

> weight: 800

Reviewer: ```suggestion
weight: 300
```

---

# Labels and Detected fields

{{% admonition type="caution" %}}
Explore Metrics is currently in [public preview](/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.
{{% /admonition %}}

> Explore Logs shows you the volume of logs with specific Labels and Detected fields.

Reviewer: ```suggestion
Explore Logs shows you the volume of logs with specific labels and detected fields.
```

Labels are attached to your log lines when they are collected from the source.

Detected fields are automatically extracted from the log text.

> You can click **Select** on a Label or Detected field to access a breakdown of its values, seeing the log volumes visualized along the way.

Reviewer: ```suggestion
You can click **Select** on a label or detected field to access a breakdown of its values, seeing the log volumes visualized along the way.
```

This experience is useful for understanding the traits of your system, and for spotting spikes or other changes.

## Guide

To explore labels with your own data, follow these steps:

> 1. Open **Explore Logs**

Reviewer: ```suggestion
1. In the main menu, select **Explore > Logs**.
```

> 2. Select a **Service** and click **Select**

Reviewer: ```suggestion
1. Click the **Select** button for the service you want to explore.
```

> 3. Click on the **Labels** tab

Reviewer: ```suggestion
1. Click the **Labels** tab.
```

> 4. Browse the labels detected for this service

Reviewer: ```suggestion
1. Browse the labels detected for this service.
```

> 5. Look for an interesting value and click **Select**

Reviewer: ```suggestion
5. Look for an interesting label and click the **Select** button.
```
